Revamped Overview Articles (#1131)
telerik/blazor-docs: components/notification/overview.md

- [NotificationModel Class](#notificationmodel-class)
- [Show Method](#show-method)

> ## Creating Notification

Reviewer: ```suggestion
## Creating Blazor Notification
```

1. Add the `<TelerikNotification>` tag to your razor page.

> 1. Set the components reference (`@ref=`).

Reviewer: ```suggestion
1. Obtain the component reference via `@ref`.
```

You can find more information on opening, closing and hiding the Notification in the [Open, Close and Hide]({%slug notification-open-close-hide%}) article.

>caption Get a reference to the Notification and use the Show method

````CSHTML
@* The fully qualified class name of the notification component so you can use its reference *@

<TelerikButton OnClick="@AutoCloseNotification">Add auto close notification</TelerikButton>

<TelerikNotification @ref="@NotificationReference"></TelerikNotification>

@code {
    Telerik.Blazor.Components.TelerikNotification NotificationReference { get; set; }

    void AutoCloseNotification()
    {
        NotificationReference.Show(new NotificationModel()
        {
            Text = "Auto Closable Notification",
            ThemeColor = "primary",
            Closable = false
        });
    }
} 
````

## Stacked Notifications

Multiple Notifications can stack if they derive from different references. [Read the Stacked Notifications article for more information...]({%slug notification-stacked-notifications%})

## Styling and Appearance

> You can customize the styling and the appearance of the Notification. [Read the Appearance article for more information...]({%slug notification-appearance%})

Reviewer: Feature sections serve 3 purposes - 
* hint about component capabilities
* improve SEO
* direct to other pages for more information

Brief one-sentence sections may not bring enough value to the reader and can make the documentation look "auto generated". It takes very little to improve all 3 roles by using relevant key words in the link and mention some technical aspects of the feature.

Also, please add a section for the Templates.

```suggestion
You can [customize the styling and the appearance of the Notification]({%slug notification-appearance%}). For example, select the animation type or choose a suitable background color.
```

>tip @[template](/_contentTemplates/notification/templates.md#one-instance-per-app-link)

## Templates

You can control the rendering of the Notification by using Templates. [See the Templates article for more information...]({%slug notification-templates%})

> ## Styling and Appearance

Reviewer: Feature sections should precede the Parameters sections. This one about Styling duplicates one above (Success, Info, Warning, Error Notifications), which in turn is too detailed and duplicates content in the Appearance article. Consider refactoring and optimization.
